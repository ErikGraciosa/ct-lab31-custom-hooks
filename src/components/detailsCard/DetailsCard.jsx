import React from 'react';

function DetailsCard() {
  return (
    <div>
      <p>Details will go here</p>
    </div>
  );
}

export default DetailsCard;
