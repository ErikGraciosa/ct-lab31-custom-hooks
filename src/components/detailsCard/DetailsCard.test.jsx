import React from 'react';
import { render, cleanup } from '@testing-library/react';
import DetailsCard from './DetailsCard';

describe('DetailsCard component', () => {
  afterEach(() => cleanup());
  it('renders DetailsCard', () => {
    const { asFragment } = render(<DetailsCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});
