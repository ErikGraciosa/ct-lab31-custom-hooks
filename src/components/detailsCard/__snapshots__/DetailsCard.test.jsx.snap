// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsCard component renders DetailsCard 1`] = `
<DocumentFragment>
  <div>
    <p />
    <img />
    <p>
      Species: 
    </p>
    <p>
      Status: 
    </p>
  </div>
</DocumentFragment>
`;
